packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -0,0 +1,176 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import autobind from 'autobind-decorator'
+import themed from 'react-themed'
+import { GMAP_EVENTS } from './utils/const'
+import { setupEvents, removeEvent } from './utils/mapEventHelpers'
+
+const enableControls = (enable = true) => ({
+  draggable: enable,
+  zoomControl: enable,
+  scrollwheel: enable,
+  disableDoubleClickZoom: enable,
+})
+
+@themed(['Gmap_FreeDrawTootip'], { pure: true })
```

Comment:
This name should be the same as the component.

Reply:
There is a separate pr for this tool top